### DISCUSSION TITLE
Does detection speed depend on the number of objects in the scene?

### DISCUSSION THREAD

**Discussion by DSA101** [ultralytics/yolov5#6095] [Q&A]
Does Yolov5 detection speed depend on the number of objects in the scene? E.g. if I am detecting cars, how the detection speed will be impacted if there is one vs 50 cars in the picture? Thanks!

**Comment by glenn-jocher** [comment]
@DSA101 no, inference speed is not correlated with the input image. 

NMS time does scale linearly with the number of objects in the image however.
